cmd/startup/startup.go

```diff
@@ -0,0 +1,308 @@
+package main
+
+import (
+	"context"
+	"crypto/rsa"
+	"crypto/x509"
+	"flag"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/api/validate"
+	"github.com/openshift/openshift-azure/pkg/arm"
+	"github.com/openshift/openshift-azure/pkg/cluster"
+	"github.com/openshift/openshift-azure/pkg/tls"
+	azureclientstorage "github.com/openshift/openshift-azure/pkg/util/azureclient/storage"
+	"github.com/openshift/openshift-azure/pkg/util/configblob"
+	"github.com/openshift/openshift-azure/pkg/util/log"
+	"github.com/openshift/openshift-azure/pkg/util/template"
+)
+
+var (
+	logLevel         = flag.String("loglevel", "Debug", "valid values are Debug, Info, Warning, Error")
+	runningUnderTest = flag.Bool("runningundertest", false, "TestConfig.RunningUnderTest")
```

Comment:
shouldn't need this? use cs.Config.RunningUnderTest instead.

Reply:
done